debian/preinst

```diff
@@ -1,8 +1,12 @@
 #!/bin/sh -e
 
 case "$1" in
-    configure)
-    adduser --system --group --no-create-home --quiet plight || true
+    install)
+    adduser --system --group --no-create-home --home /var/lib/plight --quiet plight || true
+    ;;
+
+    upgrade)
+    usermod -d /var/lib/plight plight || true
```

Comment:
So since the deb packages have never been distributed really with the wrong homedir, i dont know that i care about trying to fix it in the package